pages/Ethereum/IdentityFactory.md

```diff
@@ -10,7 +10,7 @@ The least expensive way to create a new identity is through an identity factory.
 Official contracts will provide one or more of these standard interfaces to easily generate an identity with different upgrade paths.
 
 **Remember:** Using a factory or even a proxy is just an optimization and *NOT* required.
-Any contract that matches the [DSNP Identity](/Identity/Overview) interfaces is valid.
+Any contract that matches the [DSNP Identity](/Ethereum/Identity) interfaces is valid.
```

Comment:
This seems to contradict the earlier assertion that an identity must be produced by the official Identity Factory (unless the deal is that the official factory has some particular, special signing key or something, but I'd be surprised by that in a decentralized system).

Reply:
We shouldn't be asserting anywhere that an identity must be produced by the official Identity Factory. The only other requirement is that it must be registered by the official Registry contract. @kfogel do you recall where you saw that assertion so we can fix it?

Reply:
Found it. Comments there.